UPGRADING_2.2_TO_2.8.md

```diff
@@ -41,6 +41,14 @@ DROP TABLE issue;
 COMMIT;
 ```
 
+
+### Remove unused extensions from configuration
+
+Within app VM, edit `/etc/ckan/default/production.ini`.
+Remove `geojson_preview` and `wms_preview` from the line for `ckan.plugins`.
```

Comment:
These instructions are geared towards what we'll need to do for the production deployment, where we'll have a new instance provisioned from scratch.  This config editing step only applies to instances that were initially provisioned with the old extension, so I don't think we need to include it here.